cypress/lib/helper.js

```diff
@@ -1,16 +1,10 @@
 export const booking1 = `Name: Booking 1\
-Phone: 0213212341\
+Phone: 0123456789\
 Mail: booking1@gmail.com\
 Starts at: 10.02.2019, 20:00:00\
 Ends at: 10.02.2020, 21:00:00`
 
-export const booking2 = `Name: Booking 2\
-Phone: 0213522341\
-Mail: booking2@gmail.com\
-Starts at: 10.02.2017, 20:00:00\
-Ends at: 10.02.2028, 21:00:00`
-
-export const booking2Details = `ID: booking2\
+export const booking1Details = `ID: A-1\
```

Comment:
we shouldn't have this variable. We should just do the comparisons field by field in tests.